gitbook-docs/market/cmd-reference/ask.md

```diff
@@ -1,54 +1,59 @@
 ---
-description: Scripts to manage ask orders
+description: Managing proof request orders
 ---
 
-# ask
+# Proof request utilities
 
-Usage
+`=nil;` prepared a [toolchain](https://github.com/NilFoundation/proof-market-toolchain)
+for interacting with the Proof Market.
+This page describes the script with utilities for proof requesters — `request_tools.py`.
 
-```
-python3 ask_tools.py --h    
-usage: ask_tools.py [-h] [--auth AUTH] {push,get} ...
+Make sure you've [signed in or signed up](user.md) before continuing.
 
-positional arguments:
-  {push,get}   sub-command help
-    push       push ask
-    get        get ask
+To see all parameters that can be passed to the script, run it with the `-h` option:
 
-options:
-  -h, --help   show this help message and exit
-  --auth AUTH  auth file
+```console
+foo@bar:~$ python3 request_tools.py -h
 
+usage: request_tools.py [-h] [--auth AUTH] [-v] {push,get} ...
 
-```
+positional arguments:
+  {push,get}     sub-command help
+    push         push request
+    get          get request
 
+options:
+  -h, --help     show this help message and exit
+  --auth AUTH    auth file
+  -v, --verbose  increase output verbosity
+```
 
+## Submitting proof request
+### Usage
 
-## Submit ask
+```console
+foo@bar:~$ python3 request_tools.py push -h
 
-```
-python3 ask_tools.py push --h                   
-usage: ask_tools.py push [-h] --cost COST --key KEY
+usage: request_tools.py push [-h] --cost COST --file FILE --key KEY \
+       [--generation_time GENERATION_TIME]
 
 options:
-  -h, --help   show this help message and exit
-  --cost COST  cost
-  --key KEY    key of the statement
-
+  -h, --help            show this help message and exit
+  --cost COST           cost
+  --file FILE           json file with public input
+  --key KEY             statement key
+  --generation_time GENERATION_TIME
```

Comment:
Next thing, we might update these help strings

Reply:
You mean make a task of updating help messages for these scripts?

Reply:
yes, along with more examples and such